colossalai/kernel/triton/rms_layernorm.py

```diff
@@ -64,5 +107,10 @@ def rms_layernorm(x, weight, eps, norm_output=None):
         num_warps = min(max(triton.next_power_of_2(N) // 256, 8), 32)
 
         # enqueue kernel
-        _rmsnorm_kernel[(M,)](x, y, weight, x.stride(0), N, eps, BLOCK_SIZE=BLOCK_SIZE, num_warps=num_warps)
-        return y
+        if residual is None:
+            _rmsnorm_kernel[(M,)](x, y, weight, x.stride(0), N, eps, BLOCK_SIZE=BLOCK_SIZE, num_warps=num_warps)
+        else:
+            _rmsnorm_with_residual_kernel[(M,)](
```

Comment:
Add a test for testing this case (_rmsnorm_with_residual_kernel). For now `test_rmsnorm_triton` only test `_rmsnorm_kernel`

Reply:
Done